routes.go

```diff
@@ -361,26 +363,22 @@ func (s *Server) getCurrentState(w http.ResponseWriter, r *http.Request) {
 	} else {
 		// for http you send to the protocol address
 		statusChannel := make(chan *channel.StatusChan, 1)
-		if s.transportType == HTTP {
```

Comment:
removing the transport Type check would affect the AMQP case?

Reply:
I had that before because I had functionality enabled only for HTTP

Reply:
I just removed all reference to transport Type